programs/create/templates/preact/template/newtab/base.css

```diff
@@ -0,0 +1,23 @@
+body {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  height: calc(100vh - 26px);
+}
+
+h1 {
+  font-size: 4.7em;
+}
+
+.preact {
+  animation: spin 5s linear infinite;
+}
+
+@keyframes spin {
```

Comment:
do you mind changing this animation to something else? otherwise just make it static

Reply:
I am now using the same animations as [Preact](https://preactjs.com/).